outline/electron/connect_linux.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script allows debugging outline-go-tun2socks directly on linux.
+# Instructions:
+# 1. Install the Outline client for Linux, connect to a server, and disconnect.
+#    This installs the outline controller service.
+# 2. $ git update-index --assume-unchanged connect_linux.sh
+#    This helps to avoid accidentally checking in your proxy credentials.
+# 3. Edit this script to add the IP, port, and password for your test proxy.
+# 4. $ ./connect_linux.sh
+# 5. Ctrl+C to stop proxying
+
+readonly PROXY_IP="..."
```

Comment:
It may be safer to assume these are exported variables or pass them as arguments to the script, preventing us from accidentally commiting the variables.

Reply:
Good idea!  I moved them to the command line.